code-challenges.test.js

```diff
@@ -65,5 +137,25 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
-
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
+
+//PseudoCode
+  //create array for all vowels
+  // create a function that examines an array at each index value
+  // test each index value T/F if = vowel
+    // .map to evaluate the array at each index 
+    // edit - use for loop
+    // use .push to return the index value of the first vowel
+    // .sort to sort elements as strings in order and return the index 0
+
+const vowels = ['a', 'e', 'i', 'o', 'u'];
+
+const firstVowel = (value) => {
+  let numberOfVowels = []
+  for(let i = 0; i <= value.length ; i++) {
+    if (vowels.indexOf(value[i]) >= 0){
+      numberOfVowels.push(i);
+    }
+  } 
+  return numberOfVowels.sort()[0]
+}
```

Comment:
Your logic looks great!  There are some things that could be removed and simplified a bit.  For instance you can simply return i on line 157 removing the need for lines 160 and 154.